src/app/ApplicationTemplate.Presentation/Framework/Review/ReviewSettings.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace ApplicationTemplate;
+
+/// <summary>
+/// The review prompt settings used for prompt conditions.
+/// </summary>
+public record ReviewSettings
+{
+	public int PrimaryActionCompleted { get; set; }
+
+	public int SecondaryActionCompleted { get; set; }
+
+	public int ApplicationLaunched { get; set; }
```

Comment:
Either add count to those or rename them in a way that matches their type. They are currently named like events.